x-pack/plugins/security_solution/public/cases/components/timeline_actions/add_to_case_action.tsx

```diff
@@ -70,9 +70,9 @@ const AddToCaseActionComponent: React.FC<AddToCaseActionProps> = ({
   } = useControl();
 
   const attachAlertToCase = useCallback(
-    (theCase: Case) => {
+    async (theCase: Case) => {
       closeCaseFlyoutOpen();
-      postComment({
+      await postComment({
```

Comment:
In vscode it's telling me that the `await` doesn't have an effect. I think we might need to change the interface for `postComment` so that it returns a `Promise<void>` maybe?

![image](https://user-images.githubusercontent.com/56361221/109029623-0881ec80-7691-11eb-880a-3f0dc6b11dc3.png)
****

Reply:
You are right!